src/config.rs

```diff
@@ -2348,6 +2349,9 @@ pub struct TiKvConfig {
     #[config(submodule)]
     pub coprocessor: CopConfig,
 
+    #[config(skip)]
+    pub coprocessor_v2: CoprocessorV2Config,
```

Comment:
Not sure if `#[config(skip)]` is right here. Is this okay? I did not quite understand what `#[config(submodule)]` is.

Reply:
`#[config(submodule)]` means to allow changing the config online via an HTTP url.

Reply:
Cool, thanks! Got it :)